### PR TITLE
Tweaked an awkward sentence

### PR DESCRIPTION
**Pull request by MontrealStacy**
## Purpose of this pull request

This small pull request (PR) tweaks a sentence with an awkward sentence structure.

## Affected DevDocs pages

-  bk-get-started-api.md

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Much nicer!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

File: src/guides/v2.3/get-started/bk-get-started-api.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ The APIs can be used to perform a wide array of tasks. For example:
 
 *  Integrate with CRM (Customer Relationship Management) or ERP (Enterprise Resource Planning) backend systems, such as Salesforce or Xero.
 
-*  Integrate with a [CMS](https://glossary.magento.com/cms) (Content Management System). At present, content tagging is not supported.
+*  Integrate with a [CMS](https://glossary.magento.com/cms) (Content Management System). Currently. content tagging is not supported.
```
Comment (hostep) [inline src/guides/v2.3/get-started/bk-get-started-api.md:42]: Shouldn't it be a `,` instead of a `.`?

Comment (dobooth) [reply src/guides/v2.3/get-started/bk-get-started-api.md:42]: Errrr,,,yes

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the branch with the code is successfully imported into the `magento-devdocs/devdocs` repository. Branch name: `imported-MontrealStacy-devdocs-9433`.

**Pull request merged**